conf/ccga_dx.config

```
@@ -13,6 +13,7 @@ params {
 
 singularity {
   enabled = true
+  runOoptions = "-B /mnt"
```

This looks like a typo! 
```suggestion
  runOptions = "-B /mnt"
```

---

Ah...oh well, should be fixed in most recent commit.